github/ci/prow/files/jobs/kubevirt/kubevirt-periodics.yaml

```diff
@@ -92,3 +92,33 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+
+- name: periodic-kubevirt-e2e-okd-4.1
+  interval: 24h
+  annotations:
+    fork-per-release: "true"
+  decorate: true
+  decoration_config:
+    timeout: 7h
+    grace_period: 5m
+  max_concurrency: 6
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror: "true"
+    preset-shared-images: "true"
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+      command:
+      - "/usr/local/bin/runner.sh"
+      - "/bin/sh"
+      - "-c"
+      - "export TARGET=okd-4.1 && automation/test.sh"
```

Comment:
You will have to checkout the kubevirt repo and enter the directory manually on the periodic job.

Reply:
Oh, yes. Thanks!